src/main/java/bral/dictionary/DictionaryFrame.java

```diff
@@ -0,0 +1,75 @@
+package bral.dictionary;
+
+import com.opencsv.exceptions.CsvValidationException;
+import javax.swing.*;
+import java.awt.*;
+import javax.swing.event.DocumentEvent;
+import javax.swing.event.DocumentListener;
+import java.io.IOException;
+import java.util.List;
+
+public class DictionaryFrame extends JFrame
+{
+
+    private final EnglishDictionary englishDictionary = new EnglishDictionary();
+
+
+    private JTextField searchField;
+
+    private JTextArea definitionsTextArea;
+
+    public DictionaryFrame() throws CsvValidationException, IOException
+    {
+        setSize(1000, 800);
+        setTitle("English Dictionary");
+
+        setDefaultCloseOperation(EXIT_ON_CLOSE);
+
+        JPanel main = new JPanel();
+        main.setLayout(new BorderLayout());
+        // tells the JFrame to use this JPanel
+        setContentPane(main);
+
+
+        searchField = new JTextField("");
+        main.add(searchField, BorderLayout.NORTH);
+
+        definitionsTextArea = new JTextArea("Definitions will show here...");
+        definitionsTextArea.setEditable(false);
+        definitionsTextArea.setLineWrap(true);
+        definitionsTextArea.setWrapStyleWord(true);
+        main.add(definitionsTextArea, BorderLayout.CENTER);
```

Comment:
I removed it because it was not doing anything I needed anymore. My original intent was so that the words would wrap around, but that was before I discovered the .setLineWrap, and now that is accomplished. But I can put it back I guess if there are words with a lot of definitions